launch/org.eclipse.cdt.docker.launcher/src/org/eclipse/cdt/internal/docker/launcher/ContainerLaunchUtils.java

```diff
@@ -82,4 +92,52 @@ public static final String toDockerVolume(Map<String, String> pMap, IPath path)
 		return rv;
 	}
 
+	/**
+	 * Pull an Docker-Image if is not locally available.
+	 *
+	 * The interface is rather bad, as it returns null on success to keep things simple at the call site.
+	 * This should probably be moved to linuxtools anyway.
+	 *
```

Comment:
If you decide to move, let me know and I'll try and fast-track it so you can update this patch prior to M3.

Reply:
Doesn't need to be moved.